subsys/net/lib/lwm2m/lwm2m_rw_senml_json.c

```diff
@@ -875,12 +875,71 @@ static int get_bool(struct lwm2m_input_context *in, bool *value)
 	return fd->value_len;
 }
 
+static void base64_url_safe_decode(uint8_t *data_buf, size_t buf_len)
+{
+	uint8_t *ptr = data_buf;
+
+	for (size_t i = 0; i < buf_len; i++) {
+		if (*ptr == '-') {
+			/* replace '-' with "+" */
+			*ptr = '+';
+		} else if (*ptr == '_') {
+			/* replace '_' with "/" */
+			*ptr = '/';
+		}
+		ptr++;
+	}
+}
+
+static int store_padded_modulo(uint16_t *padded_length, uint8_t *padded_buf, uint8_t *data_tail,
+			       uint16_t data_length)
+{
+	uint16_t padded_len = (data_length % 4);
```

Comment:
Could we declare a symbol for this `4`, it seems to be used as some buffer size across the functions?

Reply:
Ok. Base64 output length have to be 4 multiple. I will create symbol for 4.